src/components/modules/MainPage/ShortCard/StyledShortCard.js

```diff
@@ -28,7 +28,6 @@ export const CardHeader = styled.div`
 
 export const CardBottom = styled.div`
   display: flex;
-  flex:1;
   justify-content: space-between; 
```

Comment:
I think should be `justify-content: center;` and has a `gap: 2.5rem`

Reply:
![Screen Shot 2022-04-11 at 2 34 22 PM](https://user-images.githubusercontent.com/58893691/162837321-e5b57bae-63ce-4a3e-9950-5844844a0ad4.png)
This is justify-content: center; and has a gap: 2.5rem